onboarding-app/SecondViewController.swift

```diff
@@ -3,27 +3,178 @@ import UIKit
+        )
+        layoutSectionTitle()
+        layoutCard(
+            imageSystemName: "eurosign.circle.fill",
+            firstLineText: "4000 EUR available",
+            secondLineText: "Euro"
+        )
+    }
+    
+    func layoutStackView() {
+        stackView.axis = .vertical
+        stackView.spacing = 20
+        stackView.alignment = .leading
+        stackView.distribution = .equalSpacing
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        stackView.isLayoutMarginsRelativeArrangement = true
         
-        view.addSubview(label)
-        constraintsInit()
+        NSLayoutConstraint.activate([
+            stackView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            stackView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            stackView.trailingAnchor.constraint(equalTo: view.trailingAnchor)
+        ])
+        stackView.layoutMargins = UIEdgeInsets(top: 0, left: 20, bottom: 0, right: 20)
     }
     
-    func constraintsInit() {
+    func layoutCloseButton() {
+        let colorConfig = UIImage.SymbolConfiguration(hierarchicalColor: .systemGray)
+        let scaleConfig = UIImage.SymbolConfiguration(pointSize: 40)
+        let closeButtonImage = UIImage(systemName: "xmark.circle.fill")?.applyingSymbolConfiguration(colorConfig)?.applyingSymbolConfiguration(scaleConfig)
+        
+        var buttonConfig = UIButton.Configuration.plain()
+        buttonConfig.image = closeButtonImage
+        // TODO: ask why there's a horizontal margin around the image by default. Is this a default UIButton inset thing? Or is it an SFSymbol thing?
+        buttonConfig.contentInsets = .init(top: 0, leading: -20, bottom: 0, trailing: -20)
+        
+        closeButton = UIButton()
+        closeButton.configuration = buttonConfig
+        closeButton.addTarget(self, action: #selector(closeView), for: .touchUpInside)
+        
+        stackView.addArrangedSubview(closeButton)
         NSLayoutConstraint.activate([
-          label.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-          label.centerYAnchor.constraint(equalTo: view.centerYAnchor)
+            closeButton.leadingAnchor.constraint(equalTo: stackView.layoutMarginsGuide.leadingAnchor),
+            closeButton.topAnchor.constraint(equalTo: stackView.layoutMarginsGuide.topAnchor)
         ])
     }
-
+    
+    func layoutTitle() {
+        titleLabel = UILabel()
+        titleLabel.text = "Where do you want to send from?"
+        titleLabel.textColor = .black
+        titleLabel.font = .systemFont(ofSize: 30, weight: .bold)
+        titleLabel.numberOfLines = 0
+        
+        stackView.addArrangedSubview(titleLabel)
+        NSLayoutConstraint.activate([
+            titleLabel.leadingAnchor.constraint(equalTo: stackView.layoutMarginsGuide.leadingAnchor),
+            titleLabel.trailingAnchor.constraint(equalTo: stackView.layoutMarginsGuide.trailingAnchor)
```

Comment:
not sure why you need this, usually when adding to stack view we don't add constraints to child view (except if setting height/width).